src/frontend/src/binder/expr/function.rs

```diff
@@ -154,12 +154,19 @@ impl Binder {
             .into());
         }
         self.ensure_aggregate_allowed()?;
-        let inputs: Vec<ExprImpl> = f
-            .args
-            .into_iter()
-            .map(|arg| self.bind_function_arg(arg))
-            .flatten_ok()
-            .try_collect()?;
+        let inputs: Vec<ExprImpl> = if f.within_group.is_some() {
+            f.within_group
+                .iter()
+                .map(|x| self.bind_function_expr_arg(FunctionArgExpr::Expr(x.expr.clone())))
+                .flatten_ok()
+                .try_collect()?
+        } else {
+            f.args
+                .iter()
+                .map(|arg| self.bind_function_arg(arg.clone()))
+                .flatten_ok()
+                .try_collect()?
```

Comment:
should ignore the direct args here.

Reply:
All direct args and other args is in `args` here.

Reply:
The logic here is the same as before, does it need to be modified?